dev-packages/electron/native/src/ffmpeg.h

```diff
@@ -0,0 +1,85 @@
+#ifndef FFMPEG_H
+#define FFMPEG_H
+/**
+ * THIS FILE REDEFINES DATA AS RETURNED BY THE FFMPEG LIBRARY.
+ * HEADER FILES ARE NOT DISTRIBUTED IN OUR SETUP, HENCE THIS.
+ */
+
+/**
+ * https://ffmpeg.org/doxygen/trunk/avutil_8h_source.html#l00199
```

Comment:
These links look like they will not age well. Would there be a link that would point e.g. to today's version of these files, so that they still point to the same thing in 5 years? if not there, maybe on GH?

Reply:
GH was the way to go, I linked to the [FFmpeg mirror repository](https://github.com/FFmpeg/FFmpeg).